### DISCUSSION TITLE
[Esame 18 ottobre 2024] Esercizio 4

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#608] [Esami M-Z]
![4](https://github.com/user-attachments/assets/b8c1b210-7ddb-4bae-a9e7-0ff29398b6d2)

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/9e8fb6fb-df8f-4c4a-b847-3cff37853c28)

**Comment by niccolo-fato** [comment]
<img width="1008" height="863" alt="image" src="https://github.com/user-attachments/assets/3cad7a6e-043b-4326-a940-b76bc99080ee" />
